File: F1GameDataParser/Database/Dtos/WidgetDto.cs
using F1GameDataParser.Database.Entities.Widgets;

namespace F1GameDataParser.Database.Dtos
{
    public class WidgetDto : BaseDto
    {
        public int OverlayId { get; set; }
        public WidgetType WidgetType { get; set; }
        public int PositionX { get; set; } = 0;
        public int PositionY { get; set; } = 0;
    }
}

File: F1BroadcastTools.Client/src/app/shared/dtos/widget.dto.ts
import { BaseDto } from "./base.dto";

export enum WidgetType {
    TimingTower = 1,
    Stopwatch,
    Minimap, 
    HaloHUD
}

export interface WidgetDto extends BaseDto {
    overlayId: number;
    widgetType: WidgetType;
    positionX: number;
    positionY: number;
}